### ISSUE TITLE
Create Actions

### ISSUE THREAD

**Issue by jonathanferrari** [cal-icor/textbook#5]
Create github actions for the following purposes:
- [x] If notebooks are updated in either nb branch or nb repo, convert to `myst` and then to html and publish

**Comment by jonathanferrari** [comment]
Removed accessibility actions from the checklist; should be a separate task. Will need to be completed by the next maintainer of the book.
